src/library/dao/playlistdao.h

```diff
@@ -141,6 +141,7 @@ class PlaylistDAO : public QObject, public virtual DAO {
     QMultiHash<TrackId, int> m_playlistsTrackIsIn;
     AutoDJProcessor* m_pAutoDJProcessor;
     DISALLOW_COPY_AND_ASSIGN(PlaylistDAO);
+    void createTablePositionList(QList<int> &, QList<int> &, const int);
```

Comment:
Pass-by-value: Remove "const" qualifier
Out params: Already mentioned, and wrong coding style (space between type)
Please add parameter names to improve readability (even if one of the out params will be removed)